source/mainnet/net/resources/glossary.rst

```diff
@@ -324,17 +324,21 @@ See also our `whitepaper`_ for more details on the terms described below.
 
       See :term:`round`.
 
-      In the blockchain, time is divided into equally sized units called *slots*. On the testnet the duration of slot is one second. In every slot, each baker checks locally whether they won the lottery, which entitles the winner to bake a block in that slot. Zero, one, or multiple bakers can win the lottery. The probability of these different events is controlled by the difficulty parameter *f*. For example, with difficulty 0.5 on average every second slot will have a lottery winner.
+      In the blockchain, time is divided into equally sized units called *slots*. On the testnet the duration of slot is one second. In every slot, each validator checks locally whether they won the lottery, which entitles the winner to produce a block in that round. Zero, one, or multiple validators can win the lottery. The probability of these different events is controlled by the difficulty parameter *f*. For example, with difficulty 0.5 on average every second slot will have a lottery winner.
```

Comment:
You have changed slot to round, but this section is about slot :thinking:

Reply:
I moved part of that information up to round, so now all that is in the definition of slot is to see round instead. Please check "round."